src/crimson/admin/admin_socket.cc

```diff
@@ -0,0 +1,896 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2011 New Dream Network
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
```

Comment:
The license bit here isn't needed (though the vim and emacs mode lines are widely appreciated).

Reply:
What license header, if any, should we use in new files?
(and it's even more problematic when creating a new file from contents factored out
from an existing file with its own old copyright message).

Reply:
see https://github.com/ceph/ceph/blob/719bff9f35b9ccf272d754058a497030df8c2366/CodingStyle#L73-L74